hummingbot/client/config/config_helpers.py

```diff
@@ -169,7 +169,7 @@ def get_erc20_token_addresses() -> Dict[str, List]:
     address_file_path = TOKEN_ADDRESSES_FILE_PATH
     token_list = {}
 
-    resp = requests.get(token_list_url, timeout=3)
+    resp = requests.get(token_list_url, timeout=(0.5, 0.5))
```

Comment:
Oh, i didn't know we can set multiple values for timeout. What are they for?

Reply:
That's socket connection timeout and data reception timeout